apps/crn-server/src/validation/research-output.validation.ts

```diff
@@ -139,6 +140,9 @@ const researchOutputPostRequestValidationSchema: JSONSchemaType<ResearchOutputPo
       'sharingStatus',
       'addedDate',
       'teams',
+      'methods',
```

Comment:
there is two tasks on the fe part - so when released prior to them - adding RO is going to stop working